Assets/Scripts/Tests/Common/SpessTest.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace SS3D.Tests
+{
+    /// <summary>
+    /// Base testing class, inherited by PlayModeTest and EditModeTest.
+    /// </summary>
+    [TestFixture]
+    public abstract class SpessTest
```

Comment:
Remove the `Spess` prefix, its cool but its better to avoid funny stuff. If the Test name conflicsts with something maybe try `UnitTest`